### DISCUSSION TITLE
Multiple nested elements

### DISCUSSION THREAD

**Discussion by AndreyGunderin** [acss-io/atomizer#512] [Q&A] [answered]
Hello team.
Is it possible to use couple nested elements like this: `class1_class2:h_C(red)` ?
Regards, Andrew.

**Comment by redonkulus** [comment, marked as answer]
That is not currently possible, they would have to be separated classes (e.g `class1 class2:h C(red)`.

**Reply by AndreyGunderin** [reply]
It is possible now to use this variant: `foo_D(n)` or `class2:h_C(red)`.
So only one level of nesting is possible, isn't it?

**Reply by redonkulus** [reply]
That will work, so if the class `foo` is an ancestor in the tree, then `D(n)` will get applied. 

`class2:h_C(red)` will work, when `class2` is hovered in an ancestor node, then the `color: red` will be applied. You can try this markup in the [REPL](https://acss.io/repl.html) to test it out:

```html
<div class="class2">
    <h1 class="class2:h_C(red)">Hello world!</h1>
</div>         
```
